content/tutorials/r_citations_markdown/index.Rmd

```diff
@@ -12,7 +12,7 @@ csl: https://raw.githubusercontent.com/citation-style-language/styles/master/res
 output: 
     md_document:
         preserve_yaml: true
-        variant: gfm
+        variant: gfm+definition_lists
```

Comment:
Nice! Didn't know about that

Reply:
See https://pandoc.org/MANUAL.html#extensions